README.md

```diff
@@ -60,6 +60,38 @@ Run the TypeScript linter.
 gulp tslint
 ```
 
+## Releases
+
+Increment all versions within S3Commander to new version (Ex. 0.12.1). Currently
```

Comment:
This should say something to the effect of increment the s3commander npm package.